File: python/7 kyu/244.py
# https://www.codewars.com/kata/5b180e9fedaa564a7000009a/train/python

# Passed

def solve(s):
    uppercase_letters = 0
    lowercase_letters = 0
    
    for char in s:
        if char.isupper():
            uppercase_letters += 1
        elif char.islower():
            lowercase_letters += 1
            
    if uppercase_letters > lowercase_letters:
        s = s.upper()
    else:
        s = s.lower()
        
    return s

output = solve("CODe")
print(output)

File: nodejs/7 kyu/244.js
// https://www.codewars.com/kata/58029cc9af749f80e3001e34/train/javascript

// Passed

function getNewNotes(salary, bills) {
  let remainingSalary = salary;
  for (const bill of bills) {
    remainingSalary -= bill;
  }

  remainingSalary = remainingSalary > 0 ? remainingSalary : 0;
  let fiveEuroBills = Math.floor(remainingSalary / 5);
  return fiveEuroBills;
}

const output = getNewNotes(2000, [500, 160, 400]);
console.log(output);